content/github/authenticating-to-github/working-with-ssh-key-passphrases.md

```diff
@@ -18,10 +18,10 @@ With SSH keys, if someone gains access to your computer, they also gain access t
 You can change the passphrase for an existing private key without regenerating the keypair by typing the following command:
 
 ```shell
-$ ssh-keygen -p
+$ ssh-keygen -p -f /home/<em>you</em>/.ssh/id_ed25519
 # Start the SSH key creation process
-> Enter file in which the key is (/Users/<em>you</em>/.ssh/id_rsa): <em>[Hit enter]</em>
-> Key has comment '/Users/<em>you</em>/.ssh/id_rsa'
+> Enter file in which the key is (/home/<em>you</em>/.ssh/id_ed25519): <em>[Hit enter]</em>
+> Key has comment '<em>your_email@example.com</em>'
```

Comment:
When you enter this command the output is the `Key has comment ...` line, so we can delete 2 lines here:
```suggestion
```